site/reference/fn/eval/README.md

```diff
@@ -62,24 +62,17 @@ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/ex
 ```sh
 # read the Resources from DIR, provide them to a container my-fun as input,
 # write my-fn output back to DIR
-kpt fn run DIR/ --image gcr.io/example.com/my-fn
+kpt fn eval DIR/ --image gcr.io/example.com/my-fn
 ```
 
 ```sh
 # provide the my-fn with an input ConfigMap containing `data: {foo: bar}`
-kpt fn run DIR/ --image gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
+kpt fn eval DIR/ --image gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
 ```
 
 ```sh
-# run the functions in FUNCTIONS_DIR against the Resources in DIR
-kpt fn run DIR/ --fn-path FUNCTIONS_DIR/
-```
-
-<!-- @fnRun @verifyStaleExamples-->
-```sh
-# discover functions in DIR and run them against Resource in DIR.
-# functions may be scoped to a subset of Resources -- see `kpt help fn run`
-kpt fn run DIR/
+# run the function with config in FUNCTIONS_CONFIG_PATH against the Resources in DIR
+kpt fn eval DIR/ --image gcr.io/example.com/my-fn:v1.0.0 --fn-config FUNCTIONS_CONFIG_PATH
```

Comment:
There are other references to `kpt fn run` in this file. Please do an audit everywhere

Reply:
I updated these parts of docs because they are the command help message. But I can remove it from this PR and we can update all docs at once.

Reply:
Yes, let's do all the doc changes module flags and args separately.